doctr/datasets/svhn.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import os
+from typing import Any, Callable, Dict, List, Optional, Tuple
+
+import numpy as np
+import h5py
```

Comment:
Perhaps it's worth checking the implementation of the Pytorch team that uses scipy: https://github.com/pytorch/vision/blob/main/torchvision/datasets/svhn.py

Reply:
This is another task from this set

Reply:
@fg-mindee 
We could instead of h5py also use [mat73](https://pypi.org/project/mat73/) but this would be only a nice view (MIT License with BSD License dependency h5py :sweat_smile: ) on top because under the hood it uses also h5py.

A change into Format 2 (cropped digits) makes no sense it is implemented in torchvision and tensorflow ([https://www.tensorflow.org/datasets/catalog/svhn_cropped](https://www.tensorflow.org/datasets/catalog/svhn_cropped))

So i see two options:

1. add h5py as dependency (or mat73)
2. close this PR and do not provide SVHN dataset

wdyt ? :smiley: